app/views/document_guide/edit.html.erb

```diff
@@ -12,11 +12,6 @@
       <p>
         You can do this later but this will help speed up the process.
       </p>
-      <p>
-        The following page will allow you to both upload photos of your documents
-        with your phone, as well as submit files you already have on your
-        computer or mobile phone/tablet.
-      </p>
```

Comment:
Was this a related change? I didn't see it mentioned in the PR. Just curious!

Reply:
@adarsh oh that is actually for a different card. I will remove it from this branch add to the other